src/components/Navbar.jsx

```diff
@@ -1,7 +1,51 @@
-import React from "react";
+"use client";
 
-const Navbar = () => {
-  return <div>Navbar</div>;
+import React, { useState } from "react";
+import Image from "next/image";
+import Logo from "@/public/svg/J&MLogo.svg";
+import Navbar from "react-bootstrap/Navbar";
+import Nav from "react-bootstrap/Nav";
+import Link from "next/link";
+import { items } from "@/data/Nav";
+
+const Navigation = () => {
+  const [selected, setSelected] = useState("");
+
+  return (
+    <Navbar
+      collapseOnSelect
+      fixed="top"
+      expand="lg"
+      className="w-full min-h-[15vh] shadow-sm color-white flex justify-between items-center "
+    >
+      <Navbar.Brand className="p-0">
+        <Link
+          eventkey="1"
+          className="p-0 flex items-center"
+          href="/"
+          onClick={() => setSelected("")}
+        >
+          <Image src={Logo} alt="JM-Logo" className="w-full min-h-[5vh] m-3 " />
+        </Link>
+      </Navbar.Brand>
+
```

Comment:
Look into adding a toggle function to display the 3 line icon used by importing FaBar

Look at this example { lines : 32 - 37 } : [https://github.com/acm-ucr/saf-website/blob/dev/src/components/Navigation.jsx](url)